common/djangoapps/entitlements/api/v1/tests/test_views.py

```diff
@@ -160,10 +203,27 @@ def test_get_entitlement_by_uuid(self):
         assert response.status_code == 200
 
         results = response.data
-        assert results == CourseEntitlementSerializer(entitlement).data
+        assert results == CourseEntitlementSerializer(entitlement).data and results.get('expired_at') is None
+
+    def test_get_expired_entitlement_by_uuid(self):
```

Comment:
Nit: It would be helpful if these tests included docstrings clarifying what they're testing.